core/bin/external_node/src/config/mod.rs

```diff
@@ -101,11 +101,11 @@ impl ConfigurationSource for Environment {
 /// This part of the external node config is fetched directly from the main node.
 #[derive(Debug, Deserialize)]
 pub(crate) struct RemoteENConfig {
-    pub bridgehub_proxy_addr: Option<Address>,
-    pub state_transition_proxy_addr: Option<Address>,
-    pub transparent_proxy_admin_addr: Option<Address>,
-    /// Should not be accessed directly. Use [`ExternalNodeConfig::diamond_proxy_address`] instead.
-    pub user_facing_diamond_proxy: Address,
+    pub l1_bridgehub_proxy_addr: Option<Address>,
+    pub l1_state_transition_proxy_addr: Option<Address>,
+    pub l1_transparent_proxy_admin_addr: Option<Address>,
```

Comment:
since CTM is on GW, maybe this one should be on GW too

Reply:
EN needs these 3 addresses only to expose them in `en_getEcosystemContracts` and `zks_getBridgehubContract`. This wasn't discussed explicitly but I think we should keep them returning L1 addresses. GW ecosystem addresses can be queried from gateway (CTM, DP) or are constant (BH)